pkg/clean/clean_lb_infra.go

```diff
@@ -151,6 +154,23 @@ func (s *LBInfraCleaner) cleanupInfraCerts(ctx context.Context) error {
 	return certsBuilder.PagingUpdateResources(ctx, certsSet, common.DefaultHAPIChildrenCount, s.NSXClient, nil)
 }
 
+func (s *LBInfraCleaner) cleanupInfraDomain(ctx context.Context) error {
+	store, err := s.queryNCPCreatedResources([]string{common.ResourceTypeDomain}, model.DomainBindingType(), nil)
```

Comment:
Double confirm, the "Domain" resource is created by NCP rather than nsx-operator as here it using "NCPCreatedResources" as the filter, right?